pixi-multistyle-text.ts

```diff
@@ -75,6 +77,39 @@ export default class MultiStyleText extends PIXI.Text {
 		this.dirty = true;
 	}
 
+	public setTagStyle(tag: string, style: ExtendedTextStyle): void {
+		if (tag === "default") {
+			this.textStyles[tag] = this.assign(MultiStyleText.DEFAULT_TAG_STYLE, style);
+		} else {
+			this.textStyles[tag] = this.assign({}, style);
+		}
+
+		this._style = new PIXI.TextStyle(this.textStyles["default"]);
+		this.dirty = true;
+	}
+
+	public updateTagStyle(tag: string, style: ExtendedTextStyle): void {
+		if (tag in this.textStyles) {
+			this.assign(this.textStyles[tag], style);
```

Comment:
My thought was to include `set` as something of a shorthand for `delete` and then `update`, for if you want to _entirely_ overwrite the style.  In particular, I thought that `update` would be the more common use case, but calling it `set` might give the wrong idea of what is being done to the style object.

Regarding the content replacement:

- Could you explain what the functional difference between `hasOwnProperty` and `in` would be in this case?  Since `this.textStyles` shouldn't have a prototype chain, the usual distinction between them doesn't seem to me like it should matter.

- If `hasOwnProperty` is the way to go, why `{}.hasOwnProperty.call(this.textStyles, tag)` instead of `this.textStyles.hasOwnProperty(tag)`?

- The internal `this.assign()` I believe is correct as-is.  Remember that we no longer set all of the text properties on each tag so that we can use `assign` to do inheritance when nesting tags.